x-pack/plugins/timelines/common/search_strategy/timeline/index.ts

```diff
@@ -41,6 +43,8 @@ export interface TimelineRequestBasicOptions extends IEsSearchRequest {
   defaultIndex: string[];
   docValueFields?: DocValueFields[];
   factoryQueryType?: TimelineFactoryQueryTypes;
+  entityType?: EntityType;
+  alertConsumers?: ALERTS_CONSUMERS[];
 }
```

Comment:
In follow up PR want to adjust types so that when querying for alerts, `defaultIndex` is not a prop and make `entityType` required once updates to UI are done.